packages/component-library/src/components/form/mt-button/mt-button.vue

```diff
@@ -168,255 +169,254 @@ $mt-button-transition: all 0.15s ease-out;
   user-select: none;
   margin: 0;
   position: relative;
+}
+
+.mt-button .mt-button__content {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+}
+
+.mt-button .mt-button__content--hidden {
```

Comment:
You don't need to add `.mt-button` everywhere. It's enough to just write `.mt-button__content-hidden`. This is also true for every other selector.

Reply:
Removed